.github/workflows/LabelIssue.yml

```diff
@@ -0,0 +1,68 @@
+name: Issue Labeled
+
+on:
+  issues:
+    types: ["labeled"]
+
+jobs:
+  add_to_backlog:
+    name: Log
+    runs-on: ubuntu-latest
+    # Single quotes must be used here https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals
+    # Only limited global functions are available in this context https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#functions
+    if: |
+        github.event.issue.state == 'Open'
+        && startsWith(github.event.label.name, 'Type: ')
+
+    steps:
+      # https://github.com/actions/github-script
+      - uses: actions/github-script@v4.0.2
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const BACKLOG_PROJECT = 3897364;
+            const mediaType = { previews: ['inertia'] }; // Column related APIs are in Alpha Preview. We need to set this HTTP Header to gain access.
```

Comment:
is `inertia` some sort of project codename?

Reply:
I have no idea what that means :) Looks like a project codename. It's copy/paste from their [docs](https://docs.github.com/en/rest/reference/projects#list-project-columns).